### ISSUE TITLE
Screensaver shows Fanart and Photos when Photos is selected. 

### ISSUE THREAD

**Issue by jimmyuk** [RasPlex/RasPlex#589]
Hi since I upgraded the PMS to 1.8.0 the photos screen saver has started including fanart. I could do with it just showing "Photos" like it used to from the PMS and not the fanart (youngsters seeing fan art of some of the age 15+ movies isn't ideal) I would gladly attempt a contribution to the code myself if i knew where to locate where it's querying the PMS. 

**RPi model**: Model 3
**RasPlex**: 1.8
**PMS**: FreeBSD 1.8.0.4109
**Network**: [ethernet]

**Comment by g1t-dlanor** [comment]
I agree that the problem is one that needs to be solved, but unfortunately neither of the two main programmers of the OpenPHT project are available at this time. (One has other job obligations and the other is probably still on summer vacation). It's also dubious whether or not your problem is due to a lack of adaption to new PMS API rules on the part of OpenPHT, as it could just as well be due to a simple bug in that PMS server version and its reimplementation of the access API, which then needs to be reverted by Plex Inc. Further testing would be required to determine exactly what's going on, and then one of the programmers would have to code a fix for it (if OpenPHT is causing it), or inform Plex Inc of the need to do it (if PMS is causing it).

In the meantime I can only suggest that you reconfigure your screensaver to simply dim the display instead of showing any image slideshow. At least that will prevent the screensaver from showing your children any disturbing fanart.
